### PR TITLE
Added exercise-07

### PR DESCRIPTION
**Pull request by HolQue**

### CODE REVIEW

File: 900_building_testsuites/exercise-07/config/exercise-07_config_variant1.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//**************************************************************************
+
+{
+  "WelcomeString"   : "Hello... Robot Framework is running now!",
+
+  "Maximum_version" : "0.6.0",
```
Comment (test-fullautomation) [inline 900_building_testsuites/exercise-07/config/exercise-07_config_variant1.json]: Hi Holger,
we assume that we are always compatible to old code when we make a new version,
therefore I would set the Maximum_version to 1.0.0 and increase Minimum_version to 0.6.0 for new features.
Thank you,
 Thomas

**Review by test-fullautomation — CHANGES_REQUESTED** [review]
Hi Holger,
very good!
please check my findings.
Thank you,
 Thomas

File: 900_building_testsuites/exercise-07/config/exercise-07_config_variant1.json (hunk 1 above)
Comment (HolQue) [reply 900_building_testsuites/exercise-07/config/exercise-07_config_variant1.json]: Done

**Comment by test-fullautomation** [comment]
Hi Holger,
there is actually an open point. Minimum- and Maximum-Version should be optional.
Best would be to set only the Minimum-Version and let Maximum-Version open.
This is how I did with TML.
Thank you,
 Thomas

**Review by test-fullautomation — APPROVED** [review]

**Pull request merged**